kroxylicious-docs/docs/_modules/authorization/con-authorization-model.adoc

```diff
@@ -0,0 +1,69 @@
+:_mod-docs-content-type: CONCEPT
+
+
+[id='con-authorization-model-{context}']
+= Authorization Model
+
+In Kafka, clients perform operations on resources in Kafka.
+
+The tables below lists the resource types and the operations that apply to them.
```

Comment:
```suggestion
The following tables list the resource types and the operations that apply to them.
```

Reply:
A note that the filter currently supports only topics and the plan is to support more resources would be useful. Or similar, depending on the plan

Reply:
yes, the is for other resource types to be covered too.